src/OrchardCore.Modules/OrchardCore.Sms.Azure/Drivers/AzureSettingsDisplayDriver.cs

```diff
@@ -0,0 +1,139 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.Extensions.Localization;
+using OrchardCore.DisplayManagement.Entities;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Entities;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Mvc.ModelBinding;
+using OrchardCore.Settings;
+using OrchardCore.Sms.Azure.Models;
+using OrchardCore.Sms.Azure.Services;
+using OrchardCore.Sms.Azure.ViewModels;
+
+namespace OrchardCore.Sms.Azure.Drivers;
+
+public sealed class AzureSettingsDisplayDriver : SiteDisplayDriver<AzureSmsSettings>
+{
+    private readonly IShellReleaseManager _shellReleaseManager;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IAuthorizationService _authorizationService;
+    private readonly IPhoneFormatValidator _phoneFormatValidator;
+    private readonly IDataProtectionProvider _dataProtectionProvider;
+
+    internal readonly IHtmlLocalizer H;
+    internal readonly IStringLocalizer S;
+
+    protected override string SettingsGroupId => SmsSettings.GroupId;
+
+    public AzureSettingsDisplayDriver(
+        IShellReleaseManager shellReleaseManager,
+        IHttpContextAccessor httpContextAccessor,
+        IAuthorizationService authorizationService,
+        IPhoneFormatValidator phoneFormatValidator,
+        IDataProtectionProvider dataProtectionProvider,
+        IHtmlLocalizer<AzureSettingsDisplayDriver> htmlLocalizer,
+        IStringLocalizer<AzureSettingsDisplayDriver> stringLocalizer)
+    {
+        _shellReleaseManager = shellReleaseManager;
+        _httpContextAccessor = httpContextAccessor;
+        _authorizationService = authorizationService;
+        _phoneFormatValidator = phoneFormatValidator;
+        _dataProtectionProvider = dataProtectionProvider;
+        H = htmlLocalizer;
+        S = stringLocalizer;
+    }
+
+    public override IDisplayResult Edit(ISite site, AzureSmsSettings settings, BuildEditorContext c)
+    {
+        return Initialize<AzureSettingsViewModel>("AzureSettings_Edit", model =>
+        {
+            model.IsEnabled = settings.IsEnabled;
+            model.PhoneNumber = settings.PhoneNumber;
+            model.ConnectionString = settings.ConnectionString;
+        }).Location("Content:5#Azure")
```

Comment:
Why?

Reply:
> I think everywhere you wrote "Azure SMS" on UI, it should say "Azure Communication SMS". The reason why I suggest that is because Microsoft may at some point introduce a new service that would also send SMS messages. This way we won't have any issue implementing a mother provider.

We already did that for Azure Email, I think it's fine for now